### DISCUSSION TITLE
Transformation tab is not visible in my UI. How can I resolve this issue?

### DISCUSSION THREAD

**Discussion by Aathisurya** [airbytehq/airbyte#48683] [Connector Questions]
I am using version 1.1.0, but the Transformation tab is not visible in my UI. How can I resolve this issue?
![Screenshot from 2024-11-22 15-41-15](https://github.com/user-attachments/assets/7fe8e40a-0155-44ef-a8b3-a1f41dee8707)

**Comment by marcosmarxm** [comment]
Custom Transformation was deprecated in open source. More details [here](https://github.com/airbytehq/airbyte/discussions/34860)
